src/main/scala/cr/pulsar/Publisher.scala

```diff
@@ -92,7 +99,15 @@ object Publisher {
         override def publish(msg: E): F[MessageId] = {
           val event = serialise(msg)
 
-          logAction(event)(topic.url) &> blocker.delay(prod.send(event))
+          logAction(event)(topic.url) &> blocker.delay {
+            val msg = prod.newMessage().value(event)
```

Comment:
Very minor, but the moment I read `prod` came to my mind `production`,  `pub` could be less likely to make confusion

Reply:
The Pulsar API uses "producer" :man_shrugging: that's why it's named `prod`...

```java
Producer<String> producer = client.newProducer(Schema.STRING)
        .topic("my-topic")
        .enableBatching(false)
        .create();
// 3 messages with "key-1", 3 messages with "key-2", 2 messages with "key-3" and 2 messages with "key-4"
producer.newMessage().key("key-1").value("message-1-1").send();
```

Reply:
I guess the nomenclature we have is quite inconsistent. We have `Consumer` and `Publisher` instead of `Consumer` / `Producer` or `Subscriber` / `Publisher`. I'll raise an issue to discuss this and deal with it in a different PR.

Reply:
https://github.com/cr-org/neutron/issues/27